src/components/pager/js/Table.js

```diff
@@ -436,4 +436,4 @@ var fluid_2_0_0_beta_1 = fluid_2_0_0_beta_1 || {};
         columnDefs: [] // [{key: "columnName", valuebinding: "*.valuePath", sortable: true/false}]
     });
 
-})(jQuery, fluid_2_0_0_beta_1);
\ No newline at end of file
+})(jQuery, fluid_2_0_0);
```

Comment:
We probably can take this opportunity to add the newline at the end of the file.